django/db/backends/base/references.py

```diff
@@ -0,0 +1,129 @@
+"""
```

Comment:
If each backend isn't meant to have its own `references.py`, then I'd be more inclined to put this in `db/backends/references.py` (or maybe with a filename like `ddl_references` or `sql_references`?)